compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -251,6 +251,7 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
       {
       self()->setUseSSEForSinglePrecision();
       self()->setUseSSEForDoublePrecision();
+      self()->setSupportsAutoSIMD();
```

Comment:
Would it be better to just disable that line via "#if 0" instead? I put the line in because it will be needed in the future, but there's no real need to be able to turn it on until the codegen supports SIMD. So it might be better to just disable it for now until the codegen changes are in place.